autode/plotting.py

```diff
@@ -211,23 +217,36 @@ def plot_points(zi_s, energies, ax):
 
     Arguments:
         zi_s (np.ndarray): Estimate of reaction coordinate points
-        energies (np.ndarray): len(energies) = len(zi_s)
+        energies (list(autode.plotting.Energy)): len(energies) = len(zi_s)
         ax (matplotlib.axes.Axes):
     """
-    ax.plot(zi_s, energies, ls='--', c='k', marker='o')
+    if all(isinstance(energy, Energy) for energy in energies):
+        energies_arr = np.array([energy.item() for energy in energies])
+    else:
+        energies_arr = np.array(energies, copy=True)
```

Comment:
I'm not sure what case this else is trying to catch

Reply:
was thinking that the energies list could be floats, but I guess the docstring suggests otherwise – will remove